src/utils/format.ts

```diff
@@ -0,0 +1,10 @@
+import BN from 'bn.js'
+import { fromWei } from 'web3-utils'
+
+export function formatAmount(amount?: BN): string | null {
+  if (!amount) {
+    return null
```

Comment:
Since the format function return a string, why not return an empty string here?

Reply:
It might be more comfortable what you suggest. 
I just thought that empty string doesn't need to be the representation of an undefined value. I return null in this case so we define what to do.

I actually don't mind to change it to empty string since it also evaluates to a falsy value

Reply:
I don't have a strong opinion either. Just thinking in terms of semantics and to be more strict to default to an empty string when the function is supposed to format something for output.

Reply:
I went for your approach, we can also change it one way or the other in the future. The idea is to use this function always to format amounts so we change it just in a single place. Actually this function doesn't deal yet with decimals, but that we'll come in a future PR.

https://github.com/gnosis/dex-react/commit/f9e6792d6c68508ae4972fc10b50c1ee943ddb0c